### PR TITLE
added json sample with null values.

### PR DESCRIPTION
**Pull request by chaminda**
Please review and merg.

### CODE REVIEW

**Comment by ushanib** [comment]
Hi Chaminda,

Please include a read me file with the necessary information.

Thanks
Ushani

**Comment by chaminda** [comment]
Added README.md please review and merge.

File: products/wso2_ESB/product_features/features/transports/vfs/vfs_smb/StockQuoteProxy.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                   expression="fn:concat(fn:substring-after(get-property('MessageID'), 'urn:uuid:'), '.xml')"
+                   scope="transport"/>
+         <property name="OUT_ONLY" value="true"/>
+         <property name="ClientApiNonBlocking"
+                   value="true"
+                   scope="axis2"
+                   action="remove"/>
+         <send>
+            <endpoint>
+               <address uri="vfs:file:///home/chaminda/Documents/Testing/ESB/5.0.0/runtime/beta2/vfs/out"/>
```
Comment (nu1silva) [inline products/wso2_ESB/product_features/features/transports/vfs/vfs_smb/StockQuoteProxy.xml:23]: Can we add instructions to the README where these values should change

File: products/wso2_ESB/product_features/features/transports/vfs/vfs_smb/StockQuoteProxyEncrypted.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                   expression="fn:concat(fn:substring-after(get-property('MessageID'), 'urn:uuid:'), '.xml')"
+                   scope="transport"/>
+         <property name="OUT_ONLY" value="true"/>
+         <property name="ClientApiNonBlocking"
+                   value="true"
+                   scope="axis2"
+                   action="remove"/>
+         <send>
+            <endpoint>
+               <address uri="vfs:file:///home/chaminda/Documents/Testing/ESB/5.0.0/runtime/beta2/vfs/out"/>
```
Comment (nu1silva) [inline products/wso2_ESB/product_features/features/transports/vfs/vfs_smb/StockQuoteProxyEncrypted.xml:23]: Add to README to change path

**Review by nu1silva — CHANGES_REQUESTED** [review]

**Review by nu1silva — APPROVED** [review]

**Pull request merged**